### DISCUSSION TITLE
Twitter Embeds in React Markdown

### DISCUSSION THREAD

**Discussion by VarunPotti** [remarkjs/remark#925] [Q&A] [answered]
Hey All!

I tried having twitter embeds in react markdown.

Here is my code
https://github.com/hydralite/vortex/blob/835c8190116c44b36ebf78dd1b22bc3fa65b27c0/vortex/src/Blog/DevTO/index.tsx#L88

However it seems that it fails when I try having twitter embeds.

I return a `div` right now but when I try rendering it using react-twitter-embed it does not work.

Does anyone have any solution for this?

**Comment by ChristianMurphy** [comment, marked as answer]
Welcome @VarunPotti! Sorry you ran into a spot of trouble. 

Could you share a minimal reproducible example of the issue?
https://codesandbox.io/s/github/remarkjs/.github/tree/main/sandbox-templates/react-markdown-with-create-react-app
or https://stackblitz.com/github/remarkjs/.github/tree/main/sandbox-templates/react-markdown-with-create-react-app
could be used as starters.

A few things to note from looking over the code without running it:
1. https://github.com/hydralite/vortex/blob/835c8190116c44b36ebf78dd1b22bc3fa65b27c0/vortex/src/Blog/DevTO/index.tsx#L88 is not going to apply to `<link>` tags in the head of the document, I think you intend for this to be applied to `<a>` tags that users click on?
2. You haven't imported or used `react-twitter-embed`, you need to import and use it for it to do anything.

**Reply by VarunPotti** [reply]
Hey @ChristianMurphy

Looks like it was an issue with my regex matching.
I moved it to `<a>` tag and now it works!

Thank you!
